### PR TITLE
feat: change default of smearing, rename file

### PR DESCRIPTION
**Pull request by asalzburger**
This PR sets the default smearing on P to 5% and renames the `ReconstructionBase` to `CommonReconstruction`, in order to fit with the other examples.

### CODE REVIEW

**Comment by asalzburger** [comment]
Hi @plariono - alongside what we discussed, I set for the moment the default value to 5% P smearing, something we probably should update to be dynamic at some stage.

I also renamed the file such that it fits into the general `CommonTopic` structure.

**Review by plariono — COMMENTED** [review]
Hi @asalzburger, it looks good to me, only maybe to rename the header for consistency.

File: Examples/Run/Common/src/CommonReconstruction.cpp [hunk 1]
```diff
@@ -19,6 +17,7 @@
 #include "ActsExamples/Io/Performance/TrackFitterPerformanceWriter.hpp"
 #include "ActsExamples/Io/Root/RootTrajectoryStatesWriter.hpp"
 #include "ActsExamples/Options/CommonOptions.hpp"
+#include "ActsExamples/Reconstruction/ReconstructionBase.hpp"
```
[the remark this answers is not in the document]
Comment (asalzburger) [reply Examples/Run/Common/src/CommonReconstruction.cpp:20]: Right you are ...

**Comment by robertlangenberg** [comment]
@plariono @asalzburger ping?

**Comment by beomki-yeo** [comment]
How do you think of making PRel accessible through the commend line?

**Comment by asalzburger** [comment]
Closing this one, will become obsolete with python examples.

**Pull request closed without merging**